main_page/models.py

```diff
@@ -1,3 +1,42 @@
 from django.db import models
+from django.core import validators
 
 # Create your models here.
+class MainPagePhotoSlider(models.Model):
+    img1=models.ImageField(upload_to='photos/')
```

Comment:
If it's just one entry in the DB it probably should not have been a managed entity. Especially with 5 hardcoded images here (After save() they're pretty much hardcoded into this singleton entity). I guess you could just put them somewhere in the config (upload the 5 pictures and put their location in some config), then just use these locations to display the MainPage. Less burden and less code for that.